Find a functionally equivalent code:android.widget.TextView.paste:COMMENT
Method Modifier: private     
Comment:/**
 * Paste clipboard content between min and max positions.
 */

@@ -7,15 +7,12 @@
             CharSequence paste = clip.getItemAt(i).coerceToStyledText(getContext());
             if (paste != null) {
                 if (!didFirst) {
-                    long minMax = prepareSpacesAroundPaste(min, max, paste);
-                    min = TextUtils.unpackRangeStartFromLong(minMax);
-                    max = TextUtils.unpackRangeEndFromLong(minMax);
                     Selection.setSelection((Spannable) mText, max);
                     ((Editable) mText).replace(min, max, paste);
                     didFirst = true;

